engines/account/app/models/account/user.rb

```diff
@@ -5,8 +5,13 @@ class User < ActiveRecord::Base
 
     has_secure_password
 
+    validates :first_name, :last_name, presence: true
```

Comment:
Well, we decided that when we created the model and table. If you want to do that we need to change the migration too.